cogs/commands/chatgpt.py

```diff
@@ -59,6 +59,10 @@ async def prompt(self, ctx: Context, *, message: str):
 
     @commands.hybrid_command(help=LONG_HELP_TEXT, brief=SHORT_HELP_TEXT)
     async def chat(self, ctx: Context, *, message: Optional[str] = None):
+        if is_user_banned_openAI(ctx.author):  # if user is banned error
```

Comment:
This would be neat as a decorator - discord.py already has a `@check` decorator you could use